gtda/time_series/embedding.py

```diff
@@ -397,11 +397,11 @@ class SingleTakensEmbedding(BaseEstimator, TransformerResamplerMixin):
            Boulder, 2018; `https://scholar.colorado.edu/math_gradetds/67
            <https://scholar.colorado.edu/math_gradetds/67>`_.
 
-    [4] J. A. Perea and J. Harer, "Sliding Windows and Persistence: An \
-        Application of Topological Methods to Signal Analysis"; \
-        *Foundations of Computational Mathematics*, **15**, \
-        pp. 799--838; `doi:10.1007/s10208-014-9206-z \
-        <https://doi.org/10.1007/s10208-014-9206-z>`_.
+    .. [4] J. A. Perea and J. Harer, "Sliding Windows and Persistence: An \
```

Comment:
Again, missing dots were deliberate (entry not referenced in text). It seems to me it had been like this for a while and there were never compilation errors, has this changed?

Reply:
No, but the rendering is not consistent among entries:

![image](https://user-images.githubusercontent.com/32167173/92877807-ced36700-f40b-11ea-91a2-11a9876d0662.png)

Now that you mention, I recall that we had this discussion: should i go back to the previous version?

Reply:
I see. you make a good point about formatting, it is ugly. Maybe the best solution on all counts is (like for ripser) to actually find a place to insert this citation. I was thinking to make this [4] into [2] and refer to it just after [1] in the text, what do you think?

Reply:
For the other transformers, where none of the entries is referenced (for example FlagserPersistence), I left the references as they were (without the `..`).

Reply:
> Maybe the best solution on all counts is (like for ripser) to actually find a place to insert this citation.

Yeah, that's a good idea. Also for other transformers?

Reply:
I'd say yes!

Reply:
This isn't referenced in the text yet. What do you think of what I suggested above:
> I was thinking to make this [4] into [2] and refer to it just after [1] in the text, what do you think?

Reply:
I think it's good - i applied it